webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -1330,6 +1342,7 @@ object RudderConfig extends Loggable {
     ApiVersion(12 , true) :: // rudder 6.0, 6.1
     ApiVersion(13 , true) :: // rudder 6.2
     ApiVersion(14 , false) :: // rudder 7.0
+    ApiVersion(16 , false) :: // rudder 7.2
```

Comment:
@amousset @VinceMacBuche I was going to ask to use version 15. But we could have by default one API version by rudder version, it would be easier to explain perhaps. We may still need to have some minor version with two API versions in case of some critical, incompatible bug, but it should be rare. 
WDYT ?

Reply:
on the other hand, it's strange to not have a version 15 defined, or to have it defined if it's exactly the same as 14